src/db/utils.ts

```diff
@@ -37,6 +38,7 @@ export async function populateUsers() {
         registrationExpiration: registrationExpiry,
         registrationLastNotified: null,
         visaLastNotified: null,
+        temporaryResidency: temporaryResidency,
```

Comment:
Use shorthand notation:
```suggestion
        temporaryResidency,
```